test/parallel/test-debugger-repeat-last.js

```diff
@@ -10,7 +10,7 @@ const fixture = path.join(
 
 const args = [
   'debug',
-  `--port=${common.PORT}`,
+  '--port=0',
```

Comment:
doh! you're right. To be honest, tho, `--port=0` is something that I believe *should* work. That's a separate change tho.